README.md

```diff
@@ -133,3 +133,13 @@ To drain nodes, for example, before scaling down the cluster to 6 nodes:
             resume: "{{ inventory_hostname in desired_state }}"
     ...
 
+
+CentOS 8 and OpenHPC 2
+----------------------
+
+To deploy OpenHPC 2 on CentOS 8, you must first enable the CentOS PowerTools repo
+(this ships as standard, but disabled).  To enable PowerTools:
+
+```
+sudo dnf config-manager --set-enabled PowerTools
```

Comment:
We really ought to do this via this role dont you think?

Reply:
I think so too but let's do changes to the deployed repos as part of #6 - which shouldn't be too tricky to add in due course.